src/NosCore.GameObject/Providers/InventoryService/InventoryService.cs

```diff
@@ -474,6 +474,39 @@ public InventoryItemInstance RemoveItemAmountFromInventory(short amount, Guid id
             return null;
         }
 
+        public List<InventoryItemInstance> RemoveItemAmountFromInventoryByVNum(short amount, short itemVNum)
+        {
+            var result = new List<InventoryItemInstance>();
+            var stayAmount = (ushort)amount;
```

Comment:
Why ?

Reply:
In last review, Elendan ask for it, the understand reason to me is :
Amount only can take positive value, the argument of Elendan was to change it to ushort for prevent duplication issue.
What would I do finally ???

Reply:
Not having this line at all. Two cast are useless